var/spack/repos/builtin/packages/wireshark/package.py

```diff
@@ -42,15 +42,20 @@ class Wireshark(CMakePackage):
     variant('gtk',      default=False, description='Build with gtk')
     variant('headers',  default=True, description='Install headers')
 
+    depends_on('bison',     type='build')
     depends_on('cares')
     depends_on('doxygen',   type='build')
     depends_on('flex',      type='build')
+    depends_on('git',       type='build')
     depends_on('glib')
     depends_on('gnutls')
     depends_on('libgcrypt@1.4.2:')
     depends_on('libmaxminddb')
-    depends_on('lua')
+    depends_on('libtool@2.2.2:', type='build')
+    depends_on('libpcap')
+    depends_on('lua@:5.2.99')
```

Comment:
If only Lua 5.0 to 5.2 are supported, you should also mention the minimum version:
```python
depends_on('lua@5.0:5.2')
```